WcaOnRails/app/assets/javascripts/persons.js

```diff
@@ -31,5 +33,12 @@ onPage('persons#index', function() {
 
   $table.on('load-success.bs.table', function(e, data) {
     $('#search-box i').removeClass('fa-spinner fa-spin').addClass('fa-search');
+
+    // Update params in the url.
+    var params = personsTableAjax.queryParams(); // Get region and search params.
+    params.page = $table.bootstrapTable('getOptions').pageNumber;
+    var url = location.toString();
+    url = url.replace(/persons.*/, 'persons?' + $.param(params));
```

Comment:
I'm not sure I fully understand this line. Does this assume that persons is the first url parameter?